.appveyor.yml

```diff
@@ -71,8 +71,10 @@ install:
   #
   # Update conda, then force it to NOT update itself again
   #
+  # Somehow, the update stalls for Python 3.4, so we're disabling it for now.
```

Comment:
This comment doesn't seem to match the remark below: you are not disabling the update, just removing the explicit tie to the Anaconda channel.  Either we should remove line 77 below, or correct this comment.

Reply:
Done.